packages/blade/src/components/Input/OTPInput/__tests__/OTPInput.web.test.tsx

```diff
@@ -0,0 +1,143 @@
+import userEvent from '@testing-library/user-event';
+
+import type { ReactElement } from 'react';
+import { useState } from 'react';
+import { OTPInput } from '..';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.web';
+import assertAccessible from '~src/_helpers/testing/assertAccessible.web';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+describe('<OTPInput />', () => {
+  it('should render', () => {
+    const { container } = renderWithTheme(<OTPInput label="Enter OTP" />);
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should be focussed when autoFocus flag is passed', () => {
+    const label = 'Enter OTP';
+    // eslint-disable-next-line jsx-a11y/no-autofocus
+    const { getAllByLabelText } = renderWithTheme(<OTPInput label={label} autoFocus />);
+    const allInputs = getAllByLabelText(/character/);
+    expect(allInputs[0]).toHaveFocus();
+  });
+
+  it('should have hidden input for form submissions', () => {
```

Comment:
As a future enhancement we should add tests for form submissions too, will help us catch regressions https://github.com/razorpay/blade/issues/739